### PR TITLE
feat: add collector into quick start

### LINKED ISSUE #340: add holoinsight collector into quick start
## Describe This Problem
[holoinsight-collector](https://github.com/traas-stack/holoinsight-collector) is used to receive and forward trace data

### PR DESCRIPTION
**Pull request by sw1136562366**
# Which issue does this PR close?

Closes #340

# Rationale for this change
 [holoinsight-collector](https://github.com/traas-stack/holoinsight-collector) is used to receive and forward trace data.
Holoinsight quick start example enables trace collection and storage capabilities.

# What changes are included in this PR?
- add skywalking into dockerfile
- add collector into quick start exmple
- example enables trace
- add LocalTraceExporter for trace

# Are there any user-facing changes?
`./test/scenes/scene-default/up.sh`

# How does this change test

### CODE REVIEW

File: deploy/examples/k8s/base/collector.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    
+    processors:
+      batch:
+    
+    extensions:
+      health_check:
+      pprof:
+      zpages:
+      http_forwarder_auth:
+        url: http://gateway:8080/internal/api/gateway/apikey/check
```
Comment (xzchaoo) [inline deploy/examples/k8s/base/collector.yaml:33]: Should this be 'registry.holoinsight-server' ?

File: scripts/docker/Dockerfile [hunk 2]
```diff
@@ -28,4 +28,10 @@ RUN unzip -d /home/admin/holoinsight-server-static/ /home/admin/dist.zip && rm /
 COPY --chown=admin:admin server/all-in-one/all-in-one-bootstrap/target/holoinsight-server.jar /home/admin/app.jar
 RUN echo `date` > /home/admin/build-time
 
+COPY --chown=admin:admin --from=apache/skywalking-java-agent:8.11.0-java8 /skywalking/agent /home/admin/skywalking-agent
+
+ENV SW_AGENT_NAME holoinsight-server-example
```
Comment (xzchaoo) [inline scripts/docker/Dockerfile:33]: This vars shouldn't be added here.
Them should be set in docker-compose.yaml

File: server/common/common-service/src/main/java/io/holoinsight/server/common/security/InternalWebApiInterceptor.java [hunk 3]
```diff
@@ -31,7 +28,7 @@
 public class InternalWebApiInterceptor implements HandlerInterceptor {
   static final Set<String> LOCALHOST = Sets.newHashSet("localhost", "127.0.0.1", "0:0:0:0:0:0:0:1");
   private static final Set<String> INTERNAL_WHITE_HOSTS =
-      Sets.newHashSet("gateway.holoinsight-gateway", "gateway.holoinsight-server");
+      Sets.newHashSet("gateway.holoinsight-gateway", "gateway.holoinsight-server", "server");
```
Comment (xzchaoo) [inline server/common/common-service/src/main/java/io/holoinsight/server/common/security/InternalWebApiInterceptor.java:31]: Don't add 'server' here.
Add holoinsight.security.whiteHosts properties in application.yaml

File: deploy/examples/k8s/base/collector.yaml [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    receivers:
+      holoinsight_skywalking:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:11800
+            auth:
+              authenticator: http_forwarder_auth
+        holoinsight_server:
+          http:
+            endpoint: gateway:8080
```
[the remark this answers is not in the document]
Comment (sw1136562366) [reply deploy/examples/k8s/base/collector.yaml:17]: All example pods are deployed in the same namespace, namespace can be omitted

File: deploy/examples/k8s/base/collector.yaml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    
+    exporters:
+      otlp:
+        endpoint: gateway:11800
```
[the remark this answers is not in the document]
Comment (sw1136562366) [reply deploy/examples/k8s/base/collector.yaml:21]: All example pods are deployed in the same namespace, namespace can be omitted

File: deploy/examples/k8s/base/collector.yaml (hunk 1 above)
Comment (sw1136562366) [reply deploy/examples/k8s/base/collector.yaml:33]: The apikey check api in gateway.

**Review by xzchaoo — APPROVED** [review]
LGTM

**Pull request merged**